src/Core/Snippets/Snippets.cs

```diff
@@ -190,11 +190,18 @@ public Snippet Compile(string code)
         /// - either because they have the same id, or because they previously defined an operation
         /// which is in the new Snippet - and replaces them with `newSnippet` itself.
         /// </summary>
-        private IEnumerable<Snippet> SelectSnippetsToCompile(string code)
+        private IEnumerable<Snippet> SelectSnippetsToCompile(string code, QSharpLogger logger)
         {
             var ops = Compiler.IdentifyElements(code).Select(Extensions.ToFullName).ToArray();
             var snippetsWithNoOverlap = Items.Where(s => !s.Elements.Select(Extensions.ToFullName).Intersect(ops).Any());
 
+            var entryPointPattern = @"@\s*EntryPoint\s*\(\s*\)";
```

Comment:
The only way I can see to do this is to replace via regex prior to compilation, since the compiler may throw an error. This isn't ideal, since the regex doesn't know whether what it's removing is truly an EntryPoint attribute. If there is a better suggestion for how to do this, please let me know.